app/config/config_test.yml

```diff
@@ -14,3 +14,16 @@ web_profiler:
 
 swiftmailer:
     disable_delivery: true
+
+liip_functional_test: ~
+
+# Doctrine Configuration
+# Doctrine Configuration
+doctrine:
+    dbal:
+        driver: pdo_mysql
```

Comment:
redefine here only `dbname`. It's no need to redefine driver, host, port and so on.